data/ecosystems/s/solana.toml

```diff
@@ -121,6 +121,7 @@ sub_ecosystems = [
   "Jet Protocol",
   "Jito Network",
   "Jupiter Exchange",
+  "Just Ape","
```

Comment:
typo

Reply:
also, if you add an ecosystem here, there needs to exist a toml file with the same "title". I don't think we have a toml file for Just Ape. You have 2 options:
1. Delete this change and keep only the org change below
2. Create a toml file like just-ape.toml with the org in it and remove the org link from below